aries-site/src/examples/templates/wizard/WizardValidationExample.js

```diff
@@ -355,13 +423,16 @@ const CancellationLayer = ({ onSetOpen }) => {
           pad={{ top: 'medium', bottom: 'small' }}
         >
           <Button
-            label="No, continue setup"
+            label="No, continue wizarding"
```

Comment:
Tangential, but thinking the the word "wizarding" is awkward, extraneous. Should we simplify to "No, continue" or even better, "Continue"

Consult Vicky and update Figma accordingly?

Reply:
Funny. Maia walked by while this screen was up and asked "what's wizarding?"

Reply:
I'll consult Vicky. I think Greg's intention had been to use "wizarding" as a generic term, but teams should will in that wording with language that fits the wizard purpose. For example, "Contine setup", "Continue sign up", etc.

Reply:
I've opened an issue to track the design discussions: #1302